modules/CoreSampleGameplay/src/main/java/org/terasology/coresamplegameplay/PlayerStartingInventorySystem.java

```diff
@@ -41,52 +41,72 @@
     @ReceiveEvent(components = InventoryComponent.class)
     public void onPlayerSpawnedEvent(OnPlayerSpawnedEvent event, EntityRef player) {
         BlockItemFactory blockFactory = new BlockItemFactory(entityManager);
-        // Goodie chest
+
+        EntityRef bonusChest = getBonusChest(blockFactory);
```

Comment:
Should we consider replacing this whole thing with the new capability from https://github.com/Terasology/Inventory/pull/3 ? Or I guess maybe it can't do filled chests? 🤔

Reply:
Right, the filled chests are currently missing... that's why I gave this a quick improvement before giving the `StartingInventoryComponent` the finishing touches 😉